ivadomed/scripts/extract_small_dataset.py

```diff
@@ -71,21 +71,21 @@ def extract_small_dataset(input, output, n=10, contrast_list=None, include_deriv
             each function run is independent. Flag: ``--seed``, ``-s``.
     """
     # Create output folders
-    if not os.path.isdir(output):
-        os.makedirs(output)
+    if not Path(output).is_dir():
+        Path(output).mkdir(parents=True)
     if include_derivatives:
-        oderivatives = os.path.join(output, "derivatives")
-        if not os.path.isdir(oderivatives):
-            os.makedirs(oderivatives)
-        oderivatives = os.path.join(oderivatives, "labels")
-        if not os.path.isdir(oderivatives):
-            os.makedirs(oderivatives)
-        iderivatives = os.path.join(input, "derivatives", "labels")
+        oderivatives = Path(output, "derivatives")
```

Comment:
I would more explicitly state it as either `out_derivatives` or `derivatives_out`.